pkg/plugins/plugins.go

```diff
@@ -229,7 +241,8 @@ func (scanner *PluginScanner) walker(currentPath string, f os.FileInfo, err erro
 	if f.Name() == "plugin.json" {
 		err := scanner.loadPluginJson(currentPath)
 		if err != nil {
-			log.Error(3, "Plugins: Failed to load plugin json file: %v,  err: %v", currentPath, err)
+			// XXX: Should we use plog here instead of log?
+			log.Error(3, "Plugins: Failed to load plugin %q,  err: %v", filepath.Dir(currentPath), err)
```

Comment:
Wondering why we're using `log` here instead of `plog` like everywhere else in the file.